document_url/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
+#                       Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+{
+    "name": "URL attachment",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "author": "Tecnativa, Guadaltech, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/knowledge",
+    "license": "AGPL-3",
+    "data": ["view/document_url_view.xml", "security/ir.model.access.csv"],
```

Comment:
```suggestion
    "data": ["view/document_url_view.xml"],
```

Need to remove `security/ir.model.access.csv` file to prevent Runbot error _The transient model ir.attachment.add_url (Wizard to add URL attachment) should not have explicit access rules!_